app/forms/openid_connect_authorize_form.rb

```diff
@@ -49,8 +52,11 @@ def params
       redirect_uri: redirect_uri,
       response_type: response_type,
       scope: scope.join(' '),
-      state: state
-    }
+      state: state,
+      # TODO: do NOT echo code_challenge back to the client like this
```

Comment:
create issue instead of `TODO` comment? I know I do the same thing, but @jessieay has been trying to cure me of it for the last year+.